### PR TITLE
EU International Region Limited Availability

### PR DESCRIPTION
**Pull request by EdwardAngert**
## Effect
PR includes the following changes:
- early access to Limited Availability
- manual migration link + pro services note
- view region in dashboard
- International Region

## Remaining Work
- [ ] approval from @ari-gold 
- [ ] Copy Review
- [ ] merge on 4/8

## Post Launch
To be completed by the docs team upon merge: 
- [ ] Update Status Report
- [ ] Archive from **Done** in Waffle

### CODE REVIEW

File: source/_docs/eu-region.md [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
 
-Once granted early access, our Support team will create the new site for you.
+Use the Dashboard to confirm the Pantheon Region in which the site hosted:
+
+1.  Navigate to the site Dashboard
+1.  Click **Settings**
+1.  **About Site**
+1.  Region will show either:
+    - `United States`, by default, or
+    - `European Union`, if the site is hosted in the EU Region
```
Comment (alexfornuto) [inline source/_docs/eu-region.md]: @EdwardAngert do you think an image would be appropriate here?

File: source/_docs/eu-region.md [hunk 2]
```diff
@@ -1,72 +1,135 @@
...
+### EU Data Residency
+
+Now you can run WordPress or Drupal sites on Pantheon and meet legal, regulatory, or data sovereignty requirements from the European Union.
+
+Pantheon sites running in Europe have all site resources in the EU. This includes application and database containers, distributed filesystem and request router, Redis cache servers, and Apache Solr index servers.
+
+Automated and manual backups of all site components (code, database, and files) are stored in the EU and created by job workers also running in the EU. Additionally, any database or file clones between site environments are run by EU job workers.
 
-### Details
+## Create a New Site (Required)
```
Comment (ari-gold) [inline source/_docs/eu-region.md]: let's remove "(Required)"

File: source/_docs/eu-region.md [hunk 3]
```diff
@@ -1,72 +1,135 @@
...
+   <button class="btn btn-default btn-clippy" data-clipboard-target="#terminus-new-site">Copy</button>
+   <figure><pre id="terminus-new-site"><code class="command bash" data-lang="bash">
+  terminus site:create my-eu-site-name "My EU Site Name" "WordPress" --org "My Organization Name" --region eu</code></pre></figure>
+ </div>
 
-Once granted early access, our Support team will create the new site for you.
+  ![terminus site:create my-eu-site "My EU Site" "WordPress" --org "Rachel Pantheor" --region eu](/source/docs/assets/images/create-site-eu.png)
 
-## Use Terminus to Display Sites and Regions
+## Review Site Region (Optional)
```
Comment (ari-gold) [inline source/_docs/eu-region.md]: remove "(Optional)"

File: source/_docs/eu-region.md [hunk 4]
```diff
@@ -1,72 +1,135 @@
...
 
-In the following examples, replace or assign `$SITE` with your site name or UUID.
+Use the Dashboard to confirm the Pantheon Region in which the site is hosted:
```
Comment (ari-gold) [inline source/_docs/eu-region.md]: confirm --> see

File: source/_docs/eu-region.md [hunk 5]
```diff
@@ -1,72 +1,135 @@
...
-    www.example.com
-    blog.example.com
-    test.example.com
-    dev.example.com
+The [typical way to migrate sites](/docs/migrate/) is not yet available for international migrations. To migrate an existing site from one region to another, create the new site as described above, then follow the [Manually Migrate Sites to Pantheon](/docs/migrate-manual/) guide.
 
-    We understand that domains will be routed to the EU every Tuesday, for all requests submitted by the preceding Friday, and that we will receive confirmation when the update is complete, at which time we can update DNS.
-    ```
-4. After receiving confirmation, celebrate. Your site is running in the EU!
+If you want help migrating your site between regions, our [Professional Services Migrations](https://pantheon.io/professional-services){.external} team is available for hire.
```
Comment (ari-gold) [inline source/_docs/eu-region.md]: delete "for hire."

File: source/_docs/eu-region.md [hunk 6]
```diff
@@ -1,72 +1,135 @@
...
+The output should look something like:
+
+```bash
+curl -Is https://dev-rachel-whitton-eu2.pantheonsite.io | grep x-served-by
+x-served-by: cache-ams21041-AMS, cache-jfk8127-JFK
+```
+
+Time to celebrate. Your site is running in the EU!
+
+## Frequently Asked Questions
```
Comment (ari-gold) [inline source/_docs/eu-region.md]: what do you think about moving the FAQ to the very bottom?

File: source/_docs/eu-region.md [hunk 7]
```diff
@@ -1,72 +1,135 @@
...
-    dev.example.com
+```bash
+curl -Is https://dev-rachel-whitton-eu2.pantheonsite.io | grep x-served-by
+x-served-by: cache-ams21041-AMS, cache-jfk8127-JFK
+```
 
-    We understand that domains will be routed to the EU every Tuesday, for all requests submitted by the preceding Friday, and that we will receive confirmation when the update is complete, at which time we can update DNS.
-    ```
-4. After receiving confirmation, celebrate. Your site is running in the EU!
+Time to celebrate. Your site is running in the EU!
```
Comment (alexfornuto) [inline source/_docs/eu-region.md]: Strongly suggest losing the "cheer".

File: source/_docs/eu-region.md [hunk 8]
```diff
@@ -1,72 +1,135 @@
...
 
 ## Coming Soon
 
-More EU Region features are in active development. [Contact us](https://pantheon.io/contact-us){.external} to learn more, and check this guide for updates.
+More International Region features are in active development. [Contact us](https://pantheon.io/contact-us){.external} to learn more, and check this guide for updates.
```
Comment (alexfornuto) [inline source/_docs/eu-region.md]: Not a guide.

File: source/_docs/eu-region.md [hunk 9]
```diff
@@ -1,72 +1,135 @@
...
 
 Coming soon:
 
-  - Dashboard support
-  - Automated domain routing
+  - Disaster Recovery for EU Region
+  - Region Facet on Organization Dashboard
+  - More Regions!
+    - [Fill out this survey](https://www.getfeedback.com/r/hkR9uTAJ){.external} to tell us about your needs.
```
Comment (alexfornuto) [inline source/_docs/eu-region.md]: The Survey asks questions mostly about the EU, then asks about other regions only at the end. I think that the survey should be split into two, one for the EU and one for asking about other regions, and I think that this last bullet point should be moved to below the list.

File: source/_docs/eu-region.md [hunk 10]
```diff
@@ -1,72 +1,135 @@
...
+
+## Frequently Asked Questions
+### Can I use the EU region for an existing site?
+Yes, however you must migrate your existing site to a new Site Dashboard that was configured for the EU region during creation ([as described above](#create-a-new-site)).
```
Comment (alexfornuto) [inline source/_docs/eu-region.md]: "new Site Dashboard" is not technically correct, `rm` dashboard.

File: source/_docs/eu-region.md [hunk 11]
```diff
@@ -1,72 +1,135 @@
...
+
+### When will EU Region be available for sites paid by credit card?
```
Comment (alexfornuto) [inline source/_docs/eu-region.md]: when will THE EU region, maybe?

File: source/_docs/eu-region.md [hunk 12]
```diff
@@ -1,72 +1,135 @@
...
+General Availability is planned for late 2019. Tell us more about your needs by [filling out this survey](https://www.getfeedback.com/r/hkR9uTAJ){.external}.
+
+### When will site creation through the Dashboard be available?
+General availability is planned for late 2019.
```
Comment (alexfornuto) [inline source/_docs/eu-region.md]: Aaaaand so I find the answer to a series of questions that began on L26.

File: source/_docs/eu-region.md [hunk 13]
```diff
@@ -1,72 +1,135 @@
...
+
+Contact your account owner or our [Sales team](https://pantheon.io/contact-us){.external} to learn about Pantheon's migration services or review the [relaunch procedure](/docs/relaunch/) for steps on how to migrate the site yourself.
```
Comment (alexfornuto) [inline source/_docs/eu-region.md]: Consider referencing the relaunch doc earlier where we're telling people to do a manual migration for existing sites.

**Pull request merged**